RealmSwift/Combine.swift

```diff
@@ -503,14 +544,57 @@ public enum Publishers {
         /// This publisher emits Void.
         public typealias Output = Void
 
+        private let collection: Collection
+
+        internal init(_ collection: Collection) {
+            self.collection = collection
+        }
+
+        /// Allows you to capture the `NotificationToken` produced by observing a Realm Collection.
```

Comment:
This needs a bit more description of what it's doing and why you'd want to use it. It should mention notification skipping, how it makes that possible, and when the token is actually written to the target object/keypath.